documents/research/ros2_control.md

```diff
@@ -0,0 +1,45 @@
+## Resources
+- Intro, use with Gazebo: https://www.youtube.com/watch?v=4QKsDf1c4hc
+- Hardware after Simulation: https://www.youtube.com/watch?v=4VVrTCnxvSw
+
+## Input
+- Command Velocity
+
+### ros2_control Component
+Consists of three main components:
+- Diff Drive Controller
+- Controller Manager
+- Hardware Interface
+
+- Diff Drive controller converts command velocities into req. motor velocities 
```

Comment:
Let's formalize this a bit. We could also throw in [this diagram](https://control.ros.org/master/_images/components_architecture.png) from the ros2_control documentation for clarity.